python/torch_mlir/eager_mode/torch_mlir_dispatch.py

```diff
@@ -123,7 +126,11 @@ def build_script_function(
     else:
         graph.registerOutput(node.output())
 
-    fn_name = str(node).strip()
+    # The first alternation handles non-alpha characters at the beginning.
+    # MLIR doesn't support such symbol names unless quoted, hence safer to remove.
```

Comment:
The quoting is just the textual format. Arbitrary strings should be allowed. So I don't think this should be a factor. Downstream projects might have bugs.

Reply:
addressed in the PR that this change associated with